tox-requirements.txt

```diff
@@ -0,0 +1 @@
+Django<1.12
```

Comment:
Counter-nit:  I did notice that you and others were using 2.0 for the upper range.  I know there is not supposed to be a 1.12.  But if there were, this would be more accurate.  And if there weren't, this is still correct.  I don't see any benefit of using 2.0 here instead of 1.12, and a non-zero risk.  :)